src/openrct2/peep/GuestPathfinding.cpp

```diff
@@ -887,7 +891,7 @@ namespace OpenRCT2::PathFinding
                 &peep, "Checking map element at %d,%d,%d; Type: %s; Steps: %u", loc.x >> 5, loc.y >> 5, loc.z,
                 PathSearchToString(searchResult), numSteps);
 
-            /* At this point tileElement is of interest to the pathfinding. */
+            /* At this point tileElement is of interest to the path finding. */
```

Comment:
`pathfinding` is one word. See e.g. https://en.wiktionary.org/wiki/pathfinding

Reply:
VS actually highlighted that as wrong, Are you fine with `path-finding`?

Reply:
VS probably has a limited list of words. I typically turn spellcheck off. I’m sure you can add it to its dictionary, though.

A quick Google search appears to suggest that most sources write it as “pathfinding”, and so does (almost) all of our code, so that would be my strong preference. (Though not something I’d block the PR over either.)